solr/contrib/ltr/src/java/org/apache/solr/ltr/ranking/Feature.java

```diff
@@ -207,8 +189,16 @@ public String getName() {
       return Feature.this.name;
     }
 
+    public void setNorm(Normalizer norm) {
+      this.norm = norm;
```

Comment:
The intention behind removing the norm field in the Feature class was that once constructed and initialised from the parameters the feature would not change (the norm was not one of the parameters) and when a feature is serialised into parameters then all its parameters are included in the serialisation.

How does the FilterFeature fit into that plan? A FilterFeature would never be constructed and initialised from parameters, it is instead constructed as a combination of the actual feature and a normalizer (with both the actual feature and the normalizer being constructed and initialised from parameters).

And since the FilterFeature wraps an existing feature there is no need to clone any of the features in a feature store.